apps/webapp/app/routes/magic.tsx

```diff
@@ -7,6 +7,6 @@ export async function loader({ request }: LoaderFunctionArgs) {
 
   await authenticator.authenticate("email-link", request, {
     successRedirect: redirectTo ?? "/",
-    failureRedirect: "/login",
+    failureRedirect: "/login/magic",
```

Comment:
Haven't tested this, but should work. What about any other auth strategies?

Reply:
hey @nicktrn , For GitHub when someone clicks the continue with GitHub button, the button should only allow the users if the GitHub  email matches the regs?

Reply:
Actually, leave it like this. If people want to restrict access via email whitelist, they probably shouldn't enable GitHub auth.